mteb/tasks/Retrieval/eng/ARCChallengeRetrieval.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ARCChallenge(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="ARCChallenge",
+        description="Reasoning as Retrieval (RAR-b) format: Whether Answers to Queries in Reasoning Tasks can be retrieved as top.",
```

Comment:
Would be lovely if these descriptions were more descriptive of the task (general comment)